plugins/executor.js

```diff
@@ -26,7 +27,7 @@ const buildSchemaObj = {
     jobName,
     jobState,
     jobArchived,
-    provider: Job.provider,
+    provider: Provider.provider.optional(),
```

Comment:
Hmm, it makes `provider` optional, is it supposed to be mandatory? 

![image](https://user-images.githubusercontent.com/15989893/158455651-ee01947e-b574-4083-8198-143554412eb6.png)

Reply:
Provider field can be optional 
The params under provider are not optional if provider is configured